content/docs/card/feature-card-tsx.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Integration pills
+description: A component that displays a list of services it can integrate with.
+author: shoomankhatri
+---
+
+<ComponentPreview name="card-feature-card-tsx--docs" />
```

Comment:
@ShoomanKhatri The id should be "card-integration-pills--docs"

Reply:
Also, change all the `feature-card` to `integration-pills` in your PR, including filename and text.